lib/shared/repositories/givt_repository.dart

```diff
@@ -115,4 +122,17 @@ class GivtRepositoryImpl with GivtRepository {
       {required Map<String, dynamic> body}) async {
     return apiClient.downloadYearlyOverview(body);
   }
+
+  @override
+  Future<List<MonthlySummaryItem>> fetchMonthlySummary({
+    required String guid,
+    required String fromDate,
+    required String tillDate,
+  }) async {
+    final decodedJson =
+        await apiClient.fetchMonthlySummary(guid, fromDate, tillDate);
```

Comment:
I would make this one a bit more abstract, like a map to avoid conflicts when we would split that file